src/site/content/en/blog/debug-web-vitals-in-the-field/index.md

```diff
@@ -190,20 +190,27 @@ This means you cannot make assumptions about which element or set of elements
 will be the most common LCP candidate element for a particular page. You have to
 measure it based on real-user behavior.
 
+{% Aside %}
+For more details on these differences, see: [Why lab and field data can be
+different (and what to do about it)](/lab-and-field-data-differences/)
+{% endAside %}
+
```

Comment:
Is it worth including a similar reference to https://web.dev/crux-and-rum-differences/ either here or elsewhere?